src/components/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import '../components/navbarStyles.css'
+import logoutRequest from '../services/LogoutRequest';
+
+function Navbar(props) {
+
+    const navigate = useNavigate();
+
+    const logoutUser = () => {
+        window.localStorage.setItem('blogSiteUserLoggedIn', false);
+        logoutRequest().then();
+        props.changeLoginState(false);
+        navigate('/login');
```

Comment:
Please have a look at how promises work, you should only perform logout tasks on frontend when you receive a successful response from backend.

Reply:
Done